packages/lib-user/src/hooks/usePanoptesMemberships.js

```diff
@@ -36,6 +34,6 @@ async function fetchMemberships({ query }) {
 export function usePanoptesMemberships({ authUserId, joinStatus = null, query }) {
   const joinStatusSuccess = joinStatus === asyncStates.success
 
-  const key = (query.user_id || query.user_group_id) && authUserId ? { endpoint, query, joinStatusSuccess } : null
+  const key = (query.user_id || query.user_group_id) && authUserId ? { query, joinStatusSuccess } : null
```

Comment:
What is `jointStatusSuccess` used for here? I see it in MembersListContainer too, but I'm confused about its significance in lib-user.